ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml

```diff
@@ -5605,14 +5439,14 @@ periodics:
     org: shiftstack
     repo: shiftstack-ci
   labels:
-    ci-operator.openshift.io/cloud: openstack-vexxhost
-    ci-operator.openshift.io/cloud-cluster-profile: openstack-vexxhost
+    ci-operator.openshift.io/cloud: openstack-vh-mecha-central
+    ci-operator.openshift.io/cloud-cluster-profile: openstack-vh-mecha-central
```

Comment:
This is a bit more than stated in the PR description?

Reply:
I don't know why it did that, I ran `make jobs`.

Reply:
Okay, it shuffles the list a bit and that's it.